code/src/test/pcgen/AbstractCharacterTestCase.java

```diff
@@ -197,7 +203,8 @@ protected void setUp() throws Exception
 		FactDefinition<?, String> fd =
 				BuildUtilities.createFact(context, "SpellType", PCClass.class);
 		fd.setSelectable(true);
-		SourceFileLoader.processFactDefinitions(context);
+		// Same line appears in finishLoad()
+		// SourceFileLoader.processFactDefinitions(context);
```

Comment:
Remove commented out code?

Reply:
I will, didn’t want to remove it, thought somebody might disagree with the change. That is why the comment was added.